[CI] Update NVIDIA driver to `580.82.07`

malfet opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #163111


To make CI machines capable of running CUDA-13 tests. Unfortunately, this upgrade regresses NUMBA integration, so live patch it with https://github.com/NVIDIA/numba-cuda/commit/6e08c9d08e9de59c7af28b720289debbbd384764

This fix was suggested in https://github.com/pytorch/pytorch/issues/162878#issuecomment-3288635745

malfet committed:
Update

[ghstack-poisoned]

malfet committed:
Update

[ghstack-poisoned]

malfet committed:
Update

[ghstack-poisoned]

malfet committed:
Update

[ghstack-poisoned]

huydhn commented on .github/workflows/_runner-determinator.yml:
```diff
@@ -717,18 +717,4 @@ jobs:
       - name: Get the workflow type for the current user
         id: set-condition
         run: |
-          curr_branch="${{ inputs.curr_branch }}"
```
I guess this is just a temp change to bypass the recent issue with `no-runner-experiment`?

huydhn reviewed (approved):
Stamped to unblocked, the PR needs to be cleaned up before landing

malfet committed:
Update on "[CI] Update NVIDIA driver to `580.82.07`"



To make CI machines capable of running CUDA-13 tests. Unfortunately, this upgrade regresses NUMBA integration, so live patch it with https://github.com/NVIDIA/numba-cuda/commit/6e08c9d08e9de59c7af28b720289debbbd384764


[ghstack-poisoned]

malfet committed:
Update on "[CI] Update NVIDIA driver to `580.82.07`"



To make CI machines capable of running CUDA-13 tests. Unfortunately, this upgrade regresses NUMBA integration, so live patch it with https://github.com/NVIDIA/numba-cuda/commit/6e08c9d08e9de59c7af28b720289debbbd384764

This fix was suggested in https://github.com/pytorch/pytorch/issues/162878#issuecomment-3288635745


[ghstack-poisoned]

malfet commented:
@pytorchbot revert -m "It started to fail now, but worked just fine in PR CI" -c nosignal

malfet committed:
Update

[ghstack-poisoned]
